pkg/reconciler/ingress/lister.go

```diff
@@ -139,6 +140,10 @@ func (l *gatewayPodTargetLister) listGatewayTargets(gateway *v1alpha3.Gateway) (
 			}
 			tURL.Scheme = "http"
 		case "HTTPS":
+			if server.GetTls().GetMode() == istiov1alpha3.Server_TLSOptions_MUTUAL {
+				l.logger.Infof("Skipping Server %q because HTTPS with TLS mode MUTUAL is not supported", server.Port.Name)
```

Comment:
I'd log something other than the port name?

Reply:
There isn't much else to log.
A Gateway CRd looks like:
```
spec:
  selector:
    istio: ingressgateway
  servers:
  - hosts:
    - '*'
    port:
      name: http-standard
      number: 80
      protocol: HTTP
  - hosts:
    - yolo1.com
    port:
      name: http-yolo-1
      number: 80
      protocol: HTTP
```

Reply:
Plus it is consistent with the current code, see below code.